web_src/fomantic/build/semantic.js

```diff
@@ -344,7 +344,7 @@ $.api = $.fn.api = function(parameters) {
           cancelled: function() {
             return (module.cancelled || false);
           },
-          succesful: function() {
+          successful: function() {
```

Comment:
Thanks for this PR :) The only change I am unsure of, is this one as this is copied directly from the upstream fomantic library

Reply:
Indeed, on line 351 we also have `successful`. So there will be a naming clash now.
So I think it should stay intentionally misspelled as `succesful`, or maybe it can be removed?

Reply:
I'd prefer it to stay intentionally misspelled, at least as long as it remains upstream.

Reply:
Should I revert all the upstream fomantic changes? or just this one?

Reply:
looking at the other fomantic ones, they are comments/strings but this one is the name of the function so I think reverting just this one is ok.

Reply:
Done 88f8829

Reply:
awesome. thanks again :)